src/main/resources/config/core.yml

```
@@ -135,7 +135,7 @@ protections:
     # If false, hoppers will be enabled by DEFAULT and /chopper on will disable hopper use on them
     denyHoppers: true
 
-    # Settable to private, public, donation, or off. It defines what protections are registered as when you place them
+    # Settable to private, public, donation, supply, or off. It defines what protections are registered as when you place them
```

Might as well add display here as well if we add supply. Bit mixed on whether it's TMI here though since while it is supported, it's perhaps a bit silly to make supply the default (in most cases).
```suggestion
    # Settable to private, public, donation, display, supply, or off. It defines what protections are registered as when you place them
```

---

Applied suggestion locally to reduce number of commit.

Yes, I think so too (about meaningless default protection types) :)
If you think this should be fixed you can limit settable value, but I feel there isn't much point because many admins won't be care about it.